src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -246,8 +246,11 @@ const RISCV_WHITELIST: &[(&str, Option<Symbol>)] = &[
     ("e", Some(sym::riscv_target_feature)),
 ];
 
-const WASM_WHITELIST: &[(&str, Option<Symbol>)] =
-    &[("simd128", Some(sym::wasm_target_feature)), ("atomics", Some(sym::wasm_target_feature))];
+const WASM_WHITELIST: &[(&str, Option<Symbol>)] = &[
+    ("simd128", Some(sym::wasm_target_feature)),
+    ("atomics", Some(sym::wasm_target_feature)),
+    ("nontrapping-fptoint", Some(sym::wasm_target_feature)), // TODO: Maybe None?
```

Comment:
Fow now `Some` is ok since this will cause `#[target_feature(enable = "nontrapping-fptoint")]` to be gated. That's not all that useful anyway, so it's safe to gate for now. The main thing this is otherwise doing is, unrelated to `Some`/`None`, populating the `sess.target_features` table by querying LLVM with these feature names.